### PR TITLE
[ISSUE #474] Fix ratelimit rule refresh cache when use boltdb

### LINKED ISSUE #474: 单机boltdb版本，创建的限流规则刷新到cache层逻辑不正确
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**

**Environment**
- Version: [e.g. v1.0.0]
- OS: [e.g. CentOS8]

**Additional context**

### PR DESCRIPTION
**Pull request by chuntaojun**
**Please provide issue(s) of this PR:**
Fixes #474  #475 

- Fix ratelimit rule refresh cache when use boltdb
- Fix the original unit test of the service, so that it can be run under the single-machine boltdb storage

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration
- [ ] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [x] Naming
- [ ] HealthCheck
- [x] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by shichaoyuan** [comment]
test 文件命名可以统一一下，看到提交中有 testxxx.go  xxx_test.go test_xxx.go 好几种。

**Comment by chuntaojun** [comment]
> test 文件命名可以统一一下，看到提交中有 testxxx.go xxx_test.go test_xxx.go 好几种。

test

**Review by lepdou — APPROVED** [review]
LGTM

File: store/sqldb/instance.go [hunk 1]
```diff
@@ -905,12 +905,21 @@ func batchDeleteInstanceMeta(tx *BaseTx, instances []*model.Instance) error {
 	ids := make([]interface{}, 0, len(instances))
 	builder := strings.Builder{}
 	for idx, entry := range instances {
+		if entry.FirstRegis {
```
Comment (shichaoyuan) [inline store/sqldb/instance.go:908]: FirstRegis 的逻辑没太看明白，感觉可以点儿注释。

File: service/batch/instance.go [hunk 2]
```diff
@@ -375,10 +379,10 @@ func (ctrl *InstanceCtrl) deregisterHandler(futures []*InstanceFuture) error {
 }
 
 // batchRestoreInstanceIsolate 批量恢复实例的隔离状态，以请求为准，请求如果不存在，就以数据库为准
-func (ctrl *InstanceCtrl) batchRestoreInstanceIsolate(futures map[string]*InstanceFuture) error {
+func (ctrl *InstanceCtrl) batchRestoreInstanceIsolate(futures map[string]*InstanceFuture) (map[string]struct{}, error) {
```
Comment (andrewshan) [inline service/batch/instance.go:382]: 如果定义set，一般需要用map[string]bool

File: service/batch/instance.go [hunk 3]
```diff
@@ -391,16 +395,24 @@ func (ctrl *InstanceCtrl) batchRestoreInstanceIsolate(futures map[string]*Instan
 	if id2Isolate, err = ctrl.storage.BatchGetInstanceIsolate(ids); err != nil {
 		log.Errorf("[Batch] check instances existed storage err: %s", err.Error())
 		sendReply(futures, api.StoreLayerException, err)
-		return err
+		return nil, err
 	}
+
+	diff := make(map[string]struct{})
 	if len(id2Isolate) > 0 {
```
Comment (andrewshan) [inline service/batch/instance.go:402]: 这里圈复杂度太高，改成
```
if len(id2Isolate)==0 {
   return
}

File: store/store.go [hunk 4]
```diff
@@ -75,7 +74,7 @@ func initialize(s Store) {
 	once.Do(func() {
 		if err := s.Initialize(config); err != nil {
 			fmt.Printf("[ERROR] initialize store `%s` fail: %v", s.Name(), err)
-			os.Exit(-1)
+			panic(err)
```
Comment (andrewshan) [inline store/store.go:77]: 这里不要直接panic把？

**Review by andrewshan — APPROVED** [review]
LGTM

**Pull request merged**